nodeEditor/src/components/preview/previewAreaComponent.tsx

```diff
@@ -26,78 +27,90 @@ export class PreviewAreaComponent extends React.Component<IPreviewAreaComponentP
         this._onIsLoadingChangedObserver = this.props.globalState.onIsLoadingChanged.add((state) => this.setState({isLoading: state}));
     }
 
+    componentDidMount() {
+        this.eventNMERefresh = this.eventNMERefresh.bind(this);

```

Comment:
why that?

Reply:
Because I have not been able to make the `nodeListComponent` refresh by other means...

When switching to/from "PostProcess" / "Material" modes, the `nodeListComponent` must refresh because the list of blocks depend on the mode.

I have reused the same method to have the preview area refresh too, as I don't really know how to make it refresh by other means. I guess there's always the possibility to add some observables on the global state to make this communication, but it felt wrong to do that just for a single call for refresh...

Reply:
Replaced by observables.